contracts/LimitOrder.sol

```diff
@@ -390,31 +379,16 @@ contract LimitOrder is ILimitOrder, BaseLibEIP712, SignatureValidator, Reentranc
 
         require(takerTokenOut >= _settlement.takerTokenAmount, "LimitOrder: Insufficient token amount out from protocol");
 
-        uint256 takerTokenExtra = takerTokenOut.sub(_settlement.takerTokenAmount);
-
-        // Cap taker token profit
-        uint256 takerTokenProfitCap = _mulFactor(_settlement.takerTokenAmount, profitCapFactor);
-        uint256 takerTokenProfit = takerTokenExtra > takerTokenProfitCap ? takerTokenProfitCap : takerTokenExtra;
-
-        // Calculate taker token profit for relayer
-        uint256 takerTokenProfitFee = _mulFactor(takerTokenProfit, profitFeeFactor);
-        uint256 takerTokenProfitForRelayer = takerTokenProfit.sub(takerTokenProfitFee);
-
+        uint256 takerTokenProfit = takerTokenOut.sub(_settlement.takerTokenAmount);
```

Comment:
Maybe we can rename `takerTokenProfit` to `relayerTakerTokenProfit` or `takerTokenProfitForRelayer` along with the change?
To be more specific about where the profit should go.

Reply:
Ok, renamed in 3c56458.